README.md

```diff
@@ -1 +1,14 @@
-# malden-Personal-Project-1
\ No newline at end of file
+# Project Proposal
+High Score Tracker
+[ ] List: txt w/ all current scores
+[ ] Player input: name and score
+[ ] Loop: checking where player lies in all scores
+    - If letters in score, say "invalid input" 
```

Comment:
The TODO list here should follow this format:

```
- [ ] List: txt w/ all current scores
- [ ] Player input: name and score
- [ ] Loop: checking where player lies in all scores
    - If letters in score, say "invalid input" 
```

Specifically it is missing the `-` at the beginning which is causing it to render incorrectly on the main page.